src/Microsoft.AspNetCore.SystemWebAdapters/HttpResponse.cs

```diff
@@ -45,14 +45,29 @@ public int StatusCode
             set => _response.StatusCode = value;
         }
 
-        public string StatusDescription
+        public int SubStatusCode { get; set; } = -1;
```

Comment:
Why -1? It looks like this defaulted to 0 in ASP.NET

Reply:
good catch. there was a different value that had a default as -1 and I think I got that mixed up